sonaranalyzer-dotnet/its/sources/ManuallyAddedNoncompliantIssues/XXE.ruleset

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this doesn't seem to get applied

Reply:
It might be due to the path: `<CodeAnalysisRuleSet>..\XXE.ruleset</CodeAnalysisRuleSet>` or to the fact that it's only added for debug. 
We do a Release build or Debug?

Reply:
actually any ruleset is overriden by SonarAnalyzer.Testing.ImportBefore.targets - see https://github.com/SonarSource/sonar-dotnet/blob/master/sonaranalyzer-dotnet/its/SonarAnalyzer.Testing.ImportBefore.targets#L48

I will remove any attempt

also suppressing issues inline with pragma directives doesn't work